script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -365,7 +365,8 @@ Research regarding the body is a hopeless endeavour. Theory alone will never sur
 [sha:38f28cbea70c5f041d4b11911cb23da312f9b0dd]{
 -- Page 99, Offset 30823.
 -- そんな不確かな儀式は魔術とは呼べない。それは呪いと言うのだ、ベスティーノ」
-You cannot call such a dubious rite magecraft. That's what you would call a curse, Bestino."
+// Reads better to me, what do others think?
+You cannot call such a dubious rite magecraft. That's just a curse, Bestino."
```

Comment:
agreed. would also suggest changing the order of the first sentence to 'Such a dubious rite cannot be called magecraft'